widget/button.go

```diff
@@ -519,6 +551,37 @@ func (b *Button) Text() *Text {
 	return b.text
 }
 
+func (b *Button) initText() {
```

Comment:
So this is pretty much what createWidget below is supposed to be doing. This function is a bit redundant. I could see maybe moving initText  code into createWidget and then moving the event handlers into its own function that is called by createWidget if we want to simplify?

Reply:
Isn't createWidget called before other options?
I believe we need to move it to the end of the chain to make it work, since other options are expected to populate the necessary fields.

Reply:
I removed the `Text()` option body from the multionce list, so it's executed right away.
It doesn't create widgets/containers anymore, so it should be fine.
With this change, it now makes sense to execute the initText as a part of createWidget, since by the time multionce gets executed, the necessary constructor-provided options will already be executed (since they're not part of the multionce chain anymore).